include/mbgl/style/sources/custom_vector_source.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <mbgl/style/source.hpp>
+#include <mbgl/util/geojson.hpp>
+#include <mbgl/util/geo.hpp>
+#include <mbgl/style/sources/geojson_source.hpp>
+
+namespace mbgl {
+namespace style {
+    
+class CustomVectorSource : public Source {
+public:
+    CustomVectorSource(std::string id, GeoJSONOptions options, std::function<void(uint8_t, uint32_t, uint32_t)> fetchTile);
+  
+    void setTileData(uint8_t, uint32_t, uint32_t, const mapbox::geojson::geojson&);
+    void updateTile(uint8_t, uint32_t, uint32_t);
```

Comment:
What is the difference between `setTileData` and `updateTile`? If they're distinct, can we add some documentation?